modules/ROOT/pages/introduction/intro-config-use-fd.adoc

```diff
@@ -0,0 +1,142 @@
+= Use Flow Designer to Configure a Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Flow Designer is a tool in Anypoint Design Center that enables you to create apps visually. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To use Flow Designer to configure a connector:
+
+. Configure the input source (trigger) for the flow.
+. Configure the target connector operation.
+
+[[configure-input-source]]
+== Configure the Input Source for the Flow
+
+. Log into Anypoint Platform.
+. Click *Start designing*.
+. In Design Center, click *Create new*.
+. Click *Create new application*.
+. Enter a project name and click *Create*.
+. Click *Go straight to canvas* to exit from *Let's get started*.
++
+.Clicking *Go straight to canvas* takes you to a trigger card.
+image::introduction/intro-config-fd-go-to-canvas.png[Exit from the Let's Get Started screen]
++
+. Click on the empty trigger card in the middle of the canvas to expand it.
+. Configure one of these input sources for the connector:
```

Comment:
Something that you don't explain in this topic is that you are using the term "configuration" in two different senses.
The first is in the sense of filling in the fields in the connector.
The second is in the sense of the reusable configuration that includes the connectivity info for a connector.
Look in your screenshot for HTTP Listener. At the top, there is a Configuration tab. However, there is also the gray bar in which you see this text: "Using the CloudHub HTTP configuration. Edit  Switch configurations"
You have to make clear what the difference is between these two types. In the case of the second type, if you create a reusable connection for a connector, such as Amazon SQS, you can use that configuration for each instance of the Amazon SQS connector that you add to your flow. Moreover, a configuration that is reusable within your flow might or might not contain a connection that is reusable across projects. Please see this section of the Flow Designer documentation for more info: https://docs.mulesoft.com/design-center/fd-connections